static/css/voice_clone.css

```
@@ -722,3 +722,354 @@ html[lang="zh-CN"] .register-voice-btn .register-text {
+}
+
+.silence-stat-label {
+    color: #555;
+    font-weight: 500;
+}
+
+.silence-stat-value {
+    font-weight: 700;
+    color: #333;
+    font-size: 15px;
+    font-family: 'JetBrains Mono', 'Consolas', monospace;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)